bundle-workflow/jenkins_workflow/integtest/Jenkinsfile

```diff
@@ -0,0 +1,72 @@
+pipeline {
+    agent none
+    tools {
+        jdk "JDK14"
+        maven "maven-3.8.2"
+    }
+    stages {
+        stage('Parameters') {
+            steps {
+                script {
+                    properties([
+                            parameters([
+                                    string(
+                                            defaultValue: '',
+                                            name: 's3_bucket',
+                                            trim: true
+                                    ),
+                                    string(
+                                            defaultValue: '',
+                                            name: 'opensearch_version',
+                                            trim: true
+                                    ),
+                                    string(
+                                            defaultValue: '',
+                                            name: 'build_id',
+                                            trim: true
+                                    ),
+                                    string(
+                                            defaultValue: '',
+                                            name: 'test_run_id',
+                                            trim: true
+                                    ),
+                                    string(
+                                            defaultValue: '',
+                                            name: 'architecture',
+                                            trim: true
+                                    ),
+                            ])
+                    ])
+                }
+            }
+        }
+        stage('Choose Agent Node') {
+            steps {
+                script {
+                    if ("${architecture}" == 'x64') {
+                        agentLabel = "Jenkins-Agent-al2-x64-m5xlarge"
+                    } else if ("${architecture}" == 'arm64') {
+                        agentLabel = "Jenkins-Agent-al2-arm64-m6gxlarge"
+                    } else {
+                        error("Unknown CPU architecture is provided")
+                    }
+                }
+            }
+        }
+        stage('Run Integration Tests') {
+            agent {
+                node {
+                    label "${agentLabel}"
+                }
+            }
+            steps {
+                sh "./bundle-workflow/test.sh --s3-bucket ${s3_bucket} --opensearch-version ${opensearch_version} --build-id ${build_id} --architecture ${architecture} --test-run-id ${test_run_id}"
```

Comment:
Shouldn't this have "integ-test" in the arguments?

Reply:
It should :), thanks for that. I have updated the runner later.